packages/dendron-cli/src/commands/pod.ts

```diff
@@ -163,3 +170,25 @@ export enum PodSource {
   CUSTOM = "custom",
   BUILTIN = "builtin",
 }
+
+export const handleConflict = async (conflict: Conflict) => {
+  let conflictentries = "";
+  conflict.conflictData.forEach((key) => {
+    conflictentries = conflictentries.concat(
+      `\n${key}: \nremote: ${conflict.conflictEntry.custom.social[key]}\nlocal: ${conflict.conflictNote.custom.social[key]}\n`
+    );
+  });
+  const resp = await prompts({
+    type: "text",
+    name: "conflict",
+    message: `We noticed different fields for user ${conflict.conflictNote.title}. ${conflictentries}\n
```

Comment:
might make sense for the pod itself to generate the message & possible responses. this avoids duplication but more importantly, the pod itself knows what merge operations it can handle.

```
type Pod {
  getMergeConflictText(): string
  getMergeConflictOptions(): string[],
  // undefined = ok, string = error
  validateMergeConflictResponse(): undefined|string
```